documentation/library_guide/api_for_sycl_kernels/random.rst

```diff
@@ -88,10 +88,10 @@ Random number generation is available for SYCL* device-side and host-side code.
 
 .. code:: cpp
 
+    #include <oneapi/dpl/random>
+    #include <sycl/sycl.hpp>
     #include <iostream>
```

Comment:
As I understand, usually the order of include headers assumed as:
1) oneDPL public headers
2) SYCL headers
3) STL headers